cypress/integration/createTask.spec.js

```diff
@@ -1,14 +1,34 @@
 describe('Add task', () => {
-  beforeEach(() => {
-    cy.visit('/pet/makS69UINrJFuhAuCyui')
-    cy.get('a[href="/pet/makS69UINrJFuhAuCyui/create-task"]').click()
+  before(() => {
+    cy.login()
   })
-  it('adds a new task', () => {
+
+  after(() => {
+    cy.logout()
+  })
+
+  it('goes to the create task page', () => {
+    cy.contains('FLUFFY')
+      .parent('div')
+      .parent('section')
+      .within(() => {
+        cy.get('[data-cy=pet]').click({ force: true })
+      })
+    cy.get('a[href="/pet/1YNhFhOW02GO4x7oN4Dq/create-task"]').click()
+    cy.location().should((loc) => {
+      expect(loc.pathname).to.equal('/pet/1YNhFhOW02GO4x7oN4Dq/create-task')
+    })
+  })
+
+  it('creates a new task', () => {
     cy.get('input[name="description"]').type('Random task')
     cy.get('input[name="date"]').type('2020-05-29')
     cy.get('input[name="time"]').type('08:00')
     cy.get('input[name="person"]').type('Random person')
     cy.get('[data-cy=create-task]').submit()
+  })
+
+  it('checks that this pet exists', () => {
```

Comment:
Isn't this test checking whether a task exists? In that case maybe you could instead write something like 'checks that this task exists'

Reply:
That was a typo. Thanks for catching it!